koordinates/client.py

```diff
@@ -189,6 +190,12 @@ def _raw_request(self, method, url, headers, *args, **kwargs):
         except requests.HTTPError as e:
             logger.warning("Response: %s: %s", e, r.text)
             raise exceptions.ServerError.from_requests_error(e)
+        except requests.exceptions.TooManyRedirects as e:
+            r = e.response
+            location = r.headers.get("Location")
+            raise exceptions.RedirectException(
+                f"Server responded with redirect ({r.status_code} {location}) - check you have the client host correctly configured"
```

Comment:
There are other types of redirects aside from a different host, so perhaps simplify to:

```suggestion
                f"Server responded with redirect ({r.status_code} {location})"
```

Reply:
True - I guess I assumed that the client is responsible for generating paths, the user for supplying the host. We guarantee the paths we generate will work, but the user can provide a host which has any kind of issues.

But thinking about it, this could also pick up issues like you need to log in to your wifi access point, so I'll take out the hint and let them troubleshoot as they see fit

Reply:
Done